.changeset/slimy-pets-add.md

```diff
@@ -0,0 +1,5 @@
+---
+'@microsoft/atlas-css': minor
+---
+
+added image file including object-fit-contain and object-position-top
```

Comment:
Looking for slightly clearer documentation of the change here.

Added .object-fit-contain and .object-position-top in new image related Atomics file (/src/atomics/image.scss).